Header header
int64[] toFree
int64[] toOcc